fflas-ffpack/fflas/fflas_fgemv.inl

```diff
@@ -463,6 +463,39 @@ namespace FFLAS{
         return Y;
     }
 
+    template<class Field, class Cut, class Param>
+    typename Field::Element_ptr
+    fgemv(const Field& F,
+           const FFLAS_TRANSPOSE ta,
+           const size_t m,
+           const size_t n,
+           const typename Field::Element alpha,
+           const typename Field::ConstElement_ptr A, const size_t lda,
+           const typename Field::ConstElement_ptr X, const size_t incX,
+           const typename Field::Element beta,
+           typename Field::Element_ptr Y, const size_t incY,
+           ParSeqHelper::Parallel<Cut,Param> parH){
+
+        MMHelper<Field, MMHelperAlgo::Auto, typename FFLAS::ModeTraits<Field>::value, ParSeqHelper::Parallel<Cut,Param> > pH (F,m,n,1,parH);
+        return fgemv(F, ta, m, n, alpha, A, lda, X, incX, beta, Y, incY, pH);
+    }
+
+    template<class Field>
+    typename Field::Element_ptr
+    fgemv(const Field& F,
+           const FFLAS_TRANSPOSE ta,
+           const size_t m,
+           const size_t n,
+           const typename Field::Element alpha,
+           const typename Field::ConstElement_ptr A, const size_t lda,
+           const typename Field::ConstElement_ptr X, const size_t incX,
+           const typename Field::Element beta,
+           typename Field::Element_ptr Y, const size_t incY,
+           ParSeqHelper::Sequential seqH ){
+        MMHelper<Field, MMHelperAlgo::Classic, typename FFLAS::ModeTraits<Field>::value, ParSeqHelper::Sequential > pH (F,m,n,1,seqH);
+        return fgemv(F, ta, m, n, alpha, A, lda, X, incX, beta, Y, incY);
```

Comment:
Sure but the problem is still here: you build pH but never use it.

Reply:
OK, I see, I just did a instant push, I forgot about using this in the function. I will correct this shortly.